packages/main/src/Carousel.js

```diff
@@ -472,7 +489,7 @@ class Carousel extends UI5Element {
 	}
 
 	get arrows() {
-		const showArrows = this.showNavigationArrows && isDesktop();
+		const showArrows = this._visibleNavigationArrows && this.showNavigationArrows && isDesktop();
```

Comment:
Now `_visibleNavigationArrows` and `showNavigationArrows` sound too similar.

I see that `showNavigationArrows` does more than just show the arrows - it also controls displaying the dots.
It makes more sense to rename it to `showNavigation`.